learning-development/r.qmd

```diff
@@ -554,3 +554,10 @@ Then restart R-Studio and downloads with renv should now succeed.
 
 ---
 
+## Using R with ADA/Databricks
+
+---
```

Comment:
Sorry, I also missed this extra line break too. Can we remove as it's a ## level header so has its own already

![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/fc3b91e1-32a0-4590-8215-7d256032e3e3)